### ISSUE TITLE
Error with neovim + builtin LSP support + nvim-lspconfig

### ISSUE THREAD

**Issue by avonmoll** [julia-vscode/LanguageServer.jl#853]
### Versions

LanguageServer 3.2.0
SymbolServer 5.1.0
NVIM 0.5.0-828
[nvim-lspconfig](https://github.com/neovim/nvim-lspconfig) latest (09fc944)
julia 1.4

### Error 

This error appears in the LSP log at `~/.local/share/nvim/lsp.log`

```
┌ Info: Running language server
│   env = "/Users/uuuu/.cache/nvim/lspconfig/julials/Project.toml"
│   pwd() = "/Users/uuuu/Research/turret-multi-attacker"
│   project_path = "/Users/uuuu/.julia/environments/v1.4"
└   depot_path = ""
ERROR: MethodNotFound: MethodNotFound
Stacktrace:
 [1] send_request(::JSONRPC.JSONRPCEndpoint, ::String, ::LanguageServer.ConfigurationParams) at /Users/uuuu/.julia/packages/JSONRPC/1Kq3H/src/core.jl:178
 [2] send(::JSONRPC.JSONRPCEndpoint, ::JSONRPC.RequestType{LanguageServer.ConfigurationParams,Array{Any, 1}}, ::LanguageServer.ConfigurationParams) at /Users/uuuu/.julia/packages/JSONRPC/1Kq3H/src/typed.jl:23
 [3] request_julia_config(::LanguageServerInstance, ::JSONRPC.JSONRPCEndpoint) at /Users/uuuu/.julia/packages/LanguageServer/y1ebo/src/requests/workspace.jl:84
 [4] initialized_notification(::LanguageServer.InitializedParams, ::LanguageServerInstance, ::JSONRPC.JSONRPCEndpoint) at /Users/uuuu/.julia/packages/LanguageServer/y1ebo/src/requests/init.jl:173
 [5] (::LanguageServer.var"#112#144"{LanguageServerInstance})(::JSONRPC.JSONRPCEndpoint, ::LanguageServer.InitializedParams) at /Users/uuuu/.julia/packages/LanguageServer/y1ebo/src/languageserverinstance.jl:284
 [6] dispatch_msg(::JSONRPC.JSONRPCEndpoint, ::JSONRPC.MsgDispatcher, ::Dict{String, Any}) at /Users/uuuu/.julia/packages/JSONRPC/1Kq3H/src/typed.jl:66
 [7] run(::LanguageServerInstance) at /Users/uuuu/.julia/packages/LanguageServer/y1ebo/src/languageserverinstance.jl:308
 [8] top-level scope at none:12
```

### Configuration

Here is the relevant configuration from my `~/.vimrc`

```vim
call plug#begin('~/.vim/plugged')
if has('nvim')
Plug 'neovim/nvim-lspconfig'
endif
call plug#end()
if has('nvim')
lua << EOF
	require'lspconfig'.julials.setup{}
EOF
autocmd FileType julia setlocal omnifunc=v:lua.vim.lsp.omnifunc
nnoremap <silent> <c-l>	<cmd>lua vim.lsp.buf.definition()<CR>
nnoremap <silent> K	<cmd>lua vim.lsp.buf.hover()<CR>
nnoremap <silent> gr	<cmd>lua vim.lsp.buf.references()<CR>
nnoremap <silent> g0	<cmd>lua vim.lsp.buf.document_symbol()<CR>
nnoremap <silent> S	<cmd>lua vim.lsp.buf.signature_help()<CR>
endif
```

### Misc

I'm using the default julials configfuration from nvim-lspconfig. Thus the following is executed when a julia file is opened:

```lua
cmd = {"julia", "--project=" .. environment_directory, "--startup-file=no", "--history-file=no", "-e", [[
        using Pkg;
        Pkg.instantiate()
        using LanguageServer; using SymbolServer;
        depot_path = get(ENV, "JULIA_DEPOT_PATH", "")
        project_path = dirname(something(Base.current_project(pwd()), Base.load_path_expand(LOAD_PATH[2])))
        # Make sure that we only load packages from this environment specifically.
        empty!(LOAD_PATH)
        push!(LOAD_PATH, "@")
        @info "Running language server" env=Base.load_path()[1] pwd() project_path depot_path
        server = LanguageServer.LanguageServerInstance(stdin, stdout, project_path, depot_path);
        server.runlinter = true;
        run(server);
        ]]}
```

Basically, none of the LSP functionality seems to be working. If I try to execute one of the methods I get an error message within neovim along the lines of 
`E5108: Error executing lua ...uuu/bin/share/nvim/runtime/lua/vim/lsp/callbacks.lua:338: RPC[Error] code_name = MethodNotFound, message = "method textDocument/documentSymbol is not supported by any of the servers registered for
the current buffer"`
I imagine this is a symptom of the error that's being thrown by LanguageServer.jl in the attached julia process.
Frankly, I don't know if the root cause is due to LanguageServer.jl, nvim-lspconfig, or something to do with my setup or configuration.

**Comment by avonmoll** [comment]
I ended up installing julia 1.5.3, installing neovim 0.5.0-904 (after building from source since, apparently, the CI is down), and installing from the master branch of this package. After all that, everything seems to be working fine again. Not sure what caused the original issue.
